CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ project(xdg-desktop-portal-hyprland
 )
 
 set(CMAKE_MESSAGE_LOG_LEVEL "STATUS")
+set(SYSTEMD_SERVICES ON CACHE BOOL "Install systemd service file")
```

Comment:
since we used to by default, shouldn't it stay on by default?

Reply:
well it's on by default, we'll get systemd service installed in case of
```bash
$ cmake -Bbuild
$ cmake --build build
$ cmake --install build --prefix install
```
just like before. The new feature is that we'll be able to write
```bash
$ cmake -Bbuild -DSYSTEMD_SERVICES=OFF
``` 
to skip it. Or you mean it should be OFF by default?

Reply:
ah, sorry, misread